include/psen_scan_v2/reply_msg_from_scanner.h

```diff
@@ -41,23 +41,30 @@ class ReplyMsgFromScanner
 public:
   static ReplyMsgFromScanner fromRawData(const RawScannerData& data);
 
-public:
-  ReplyMsgFromScanner(const uint32_t op_code, const uint32_t res_code);
+private:
+  template <typename T>
+  static void read(std::istringstream& is, const T& data);
 
 public:
+  ReplyMsgFromScanner(const uint32_t op_code, const uint32_t res_code);
   ReplyMsgFromScannerType type() const;
-
-public:
   static uint32_t getStartOpCode();
   static uint32_t calcCRC(const ReplyMsgFromScanner& msg);
 
+private:
+  template <typename T>
+  static void process_bytes(boost::crc_32_type& crc_32, const T& data);
+
+public:
   using RawType = std::array<char, REPLY_MSG_FROM_SCANNER_SIZE>;
   RawType toCharArray();
 
 private:
+  template <typename T>
+  void write(std::ostringstream& os, const T& data) const;
+
   ReplyMsgFromScanner() = delete;
 
-private:
```

Comment:
Why did you delete these modifiers? Of course they are obsolete in the sense that the modifier is already stated above, but they increase the readability and provide a visual separator for the reader of the class.